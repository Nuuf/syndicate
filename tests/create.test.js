/* eslint-disable @typescript-eslint/no-var-requires */
const syndicate = require('../dist');

test('creates an entity', () => {
  const entity = syndicate.create({});
  expect(JSON.stringify(entity, null, 2)).toMatchSnapshot();
});
