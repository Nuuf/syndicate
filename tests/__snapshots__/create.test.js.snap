// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates an entity 1`] = `
"{
  \\"__isSyndicateEntity__\\": true,
  \\"parentIndex\\": -1,
  \\"parentId\\": null,
  \\"index\\": -1,
  \\"id\\": \\"1\\",
  \\"data\\": {},
  \\"childrenIndices\\": [],
  \\"childrenIds\\": []
}"
`;
